scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
+SHASUM_FILE="${SCRIPT_DIR}/apollo/.shasum"
+APOLLO_DIR="${SCRIPT_DIR}"/apollo
+
+# Helper functions
+remove_existing_apollo() {
+  rm -r "${APOLLO_DIR}"
+}
+
+extract_cli() {
+  tar xvzf "${SCRIPT_DIR}"/apollo.tar.gz -C "${SCRIPT_DIR}"
+  touch "${SHASUM_FILE}"
+  tee "${SHASUM_FILE}" <<<"${SHASUM}"
+}
+
+check_shasum() {
+  # Make sure the SHASUM matches the release for this version
+  EXPECTED_SHASUM="13febaa462e56679099d81502d530e16c3ddf1c6c2db06abe3822c0ef79fb9d2  ${ZIP_FILE}"
```

Comment:
Ah, I see you're aware of the filename in the output now. 😄

Reply:
yep!